Project/Assets/ML-Agents/Examples/SharedAssets/Scripts/ModelOverrider.cs

```diff
@@ -192,69 +185,79 @@ public NNModel GetModelForBehaviorName(string behaviorName)
                 return m_CachedModels[behaviorName];
             }
 
-            string assetPath = null;
-            if (m_BehaviorNameOverrides.ContainsKey(behaviorName))
-            {
-                assetPath = m_BehaviorNameOverrides[behaviorName];
-            }
-            else if (!string.IsNullOrEmpty(m_BehaviorNameOverrideDirectory))
+            if (string.IsNullOrEmpty(m_BehaviorNameOverrideDirectory))
             {
-                assetPath = Path.Combine(m_BehaviorNameOverrideDirectory, $"{behaviorName}.{m_OverrideExtension}");
-            }
-
-            if (string.IsNullOrEmpty(assetPath))
-            {
-                Debug.Log($"No override for BehaviorName {behaviorName}, and no directory set.");
```

Comment:
Simplified the logic here now that we don't have individual overrides - now we can just check m_BehaviorNameOverrideDirectory directly.